info/guide/guide-chapter-6.md

```diff
@@ -246,7 +246,7 @@ class A {
 ```
 
 From the callee code these methods look like an access to this property: `A().isEmpty = true` for setter and `A().isEmpty` for getter.
-But in all cases it is very confusing when `get` and `set` are overriden for a developer who uses this particular class. 
+But in all cases it is very confusing when `get` and `set` are overridden for a developer who uses this particular class. 
```

Comment:
Regenerate full guide too?

Reply:
anyway, will regenerate later with the 7th chapter

Reply:
I mean, I think it's better to keep these changes consistent. Later in some pull request there will be changes in full code style, that are not reflected in individual chapter, because they have been made before in another pull request.

Reply:
I am trying to make release-model for specification. Would like to make a bunch of changes at once

Reply:
OK